calendar/calendar-tool/tool/src/webapp/vm/calendar/chef_calendar-subscriptions.vm

```diff
@@ -16,7 +16,11 @@
 			<br/>
 			<h4>$tlang.getString('subs.institutional')</h4>
 			<p class="instruction">
-				$tlang.getString('subs.institutional.instruction')
+				$tlang.getString('subs.institutional.instruction')<br/>
+				$tlang.getString('subs.institutional.instruction.detailed.1')<br/>
+				$tlang.getString('subs.institutional.instruction.detailed.2')<br/>
+				$tlang.getString('subs.institutional.instruction.detailed.3')<br/>
+				$tlang.getString('subs.institutional.instruction.detailed.4')
 			</p>
```

Comment:
No, i did it in that way but <ul> elements are not permited inside <p> elements.

Did some tests and finally i got this solution, is not the best but the text is inside the "instructions" paragraph.